content/en/blog/pop-joins-falco.md

```diff
@@ -0,0 +1,58 @@
+
+ Falco is quickly becoming the de facto runtime security tool for cloud native. Falco is powerful, it’s generic enough to be used for almost any scenario you need and flexible enough to integrate with whatever you need. Falco is being adopted by more and more firms, Shopify, Booz Allen Hamilton, Coveo, Sumo Logic, and many others.
+
+ ### POP's Plan and Focus
+
+ #### Contribution - In order to continue to add contributors, we will:
+
+ - Help explain what it means to contribute to Falco.  If you have a talent for creating Falco rulesets or you want to work on Falco core, I want to help you get there.
+
+ - Make sure folks are welcome when contributing to Falco.  We accomplish this by being available to assist in getting started, answering questions and linking contributors with subject matter experts in order to better encourage contribution
+
+ - Show gratitude for those who are key contributors to the project. Sometimes the easiest way to do this is to say “thank you” or highlight where someone’s contribution is benefiting end users and companies adopting Falco for runtime security.
+
+ - Highlight key contributions to the community (i.e Contributors of the month, Falco welcome kits for those making an impact or who have done their first PR)
+
+ - Make it fun to be part of the Falco project and celebrate how incredible this technology is!  It’s fun to contribute to something that is a great project. I want to build more relationships and foster some amazing technical breakthroughs.
+
+
+ #### Adoption -  Work with current adopters and find more end users using Falco
+
+
+ - Work within the cloud-native community to see where Falco is being used and shepherd their experience so that they also are also advocates for the power of Falco!
+
+ - Assist current adopters with Falco and ensure they have a sounding board for their integration and use of Falco so that their adoption grows even further!
+
+ #### Graduation - Help the project grow from Incubated to Graduated CNCF project
+
+ - Work under the guidance of the CNCF in order to ensure Falco is compliant with the requirements to graduate.
```

Comment:
what makes falco a graduation ready project?
it's more than just checking boxes prescribed by a dedicated marketing committee under the linux foundation.